src/library/components/molecules/as-nav-label/as-nav-label.scss

```diff
@@ -0,0 +1,146 @@
+// Component mixins
+@mixin as-nav-label-small-styles() {
+  width: 188px;
+
+  .as-nav-label {
+    &__notification-icon {
+      order: 1;
+    }
+
+    &__text {
+      max-width: 60px;
+
+      @include font-sizing-selector('button/extrasmall/regular');
+    }
+  }
+}
+
+@mixin as-nav-label-mobile-styles() {
+  position: relative;
+  width: 54px;
+  height: 54px;
+  border-radius: 54px;
+  padding: 4px;
+  display: grid;
+  place-content: center;
+
+  & .as-nav-label {
+    &__text {
+      display: none;
+    }
+
+    &__right-icon {
+      display: none;
+    }
+
+    &__left-section {
+      display: inline-block;
+    }
+
+    &__notification-icon {
+      position: absolute;
+      top: 16.8px;
+      right: 16.8px;
+    }
+  }
+
+  &.is-small {
+    width: 46px;
+    height: 46px;
+
+    & .as-nav-label__notification-icon {
+      top: 12.8px;
+      right: 12.8px;
+    }
+  }
+}
+
+// Component styles
+.as-nav-label {
+  border-radius: 4px;
+  padding: 15px 20px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 260px;
+  cursor: pointer;
+
+  &__left-section {
+    display: flex;
+    align-items: center;
+    gap: 16px;
+  }
+
+  &__left-icon {
+    color: $colors-black-60;
+    font-size: 1rem;
+  }
+
+  &__right-icon {
+    color: $colors-black-60;
+    font-size: 0.75rem;
+  }
+
+  &__right-section {
+    display: flex;
+    align-items: center;
+    gap: 16px;
+  }
+
+  &__text {
+    @include font-sizing-selector('button/small/regular');
+
+    max-width: 129px;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    color: $colors-black-60;
+  }
+
+  &:hover,
+  &.is-hover {
+    background-color: $colors-white-20;
+  }
+
+  &:active,
+  &.is-active {
+    background-color: $colors-primary-10;
+
+    .as-nav-label {
+      &__left-icon,
+      &__text,
+      &__right-icon {
+        color: $colors-primary-90;
+      }
+    }
+  }
+
+  &.is-disabled {
+    cursor: not-allowed;
+    background-color: $colors-white-40;
+
+    .as-nav-label {
+      &__left-icon,
+      &__text,
+      &__right-icon {
+        color: $colors-white-90;
+      }
+    }
+  }
+
+  &.is-small {
+    @include as-nav-label-small-styles;
+  }
+
+  @media (width <= 1024px) {
+    @include as-nav-label-small-styles;
+  }
+
+  &.is-mobile {
+    @include as-nav-label-mobile-styles;
+  }
+
+  @media (width <= 768px) {
```

Comment:
Create a mixin to avoid create a lot media querys with diferents sizes

Reply:
Done!!!